setup.py

```diff
@@ -11,10 +11,9 @@
     description='Data scrolling in the terminal',
     long_description=long_description,
     long_description_content_type='text/markdown',
-    author='Ben Ogorek',
-    author_email='baogorek@gmail.com',
+    author='Datascroller team',
```

Comment:
If we're going to go with a team name, I'd go with "The Datascroller Team." (For reference, the pandas equivalent is "The PyData Development Team"). 

Since we have the emails comma separated, we could also do "Ben Ogorek, Kevin Merfeld, and John Merfeld" as suggested [here](https://stackoverflow.com/questions/9999829/how-to-specify-multiple-authors-emails-in-setup-py). Up to you.

Reply:
Updated